ftl_installer/config/Config.py

```diff
@@ -0,0 +1,37 @@
+import yaml
+
+
+class Config(object):
+    """
+    Class to handle the configuration given with `--config` option.
+    """
+
+    def __init__(self):
+        """
+        generate an empty config when no input config file provided.
+        """
+
+        self.config = ""
+
+    def config_from_file(self, config_file):
+        """
+        stores the configuration from file to a config object.
+        """
+
+        with open(config_file) as config_file:
+            self.config = yaml.safe_load(config_file)
+
+    def write_config(self):
+        """
+        writes the config object to a file in block format.
+        """
+
+        with open('config.yml', 'w') as outfile:
```

Comment:
where should the config.yml saved. Will there be any specific place to store it?

instead of naming the config_from_file to __init__. I can use __init__ for class variable declaration and other class stuff and call the config_from_file method from it depending upon if a config file is provided. So that later on if something else is to be added to config_from_file, it remains in that only. What do you say @detiber.